docs/life_of_a_godwoken_transaction.md

```diff
@@ -206,13 +190,15 @@ While all operations on layer 2 Godwoken can be represented as a `L2Transaction`
 }
 ```
 
-This is in fact the JSON representation of the above `L2Transaction` data structure. `args` contains a variable length, free formatted transaction argument data that is interpreted depending on the values of `to_id`. For this particular example, `args` contains [SUDTTransfer](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L211-L215) data structure in molecule serialization format:
+This is in fact the JSON representation of the above `L2Transaction` data structure. `args` contains a variable length, free formatted transaction argument data that is interpreted depending on the values of `to_id`. For this particular example, `args` contains [SUDTTransfer](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L260) data structure in molecule serialization format:
```

Comment:
This is the JSON representation of the `L2Transaction` data structure. `args` contains a variable length, free formatted transaction argument data that is interpreted depending on the values of `to_id`. In this particular example, `args` contains the [SUDTTransfer](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L260) data structure in the molecule serialization format: